packages/playground/common/src/index.ts

```diff
@@ -21,7 +21,8 @@ const tmpPath = '/tmp/file.zip';
 export const unzipFile = async (
 	php: UniversalPHP,
 	zipPath: string | File,
-	extractToPath: string
+	extractToPath: string,
+	overwrite = true
```

Comment:
Let's rename this to:

```suggestion
	overwriteFiles = true
```

Otherwise it's unclear what happens when a directory name conflicts – e.g. is the old directory wiped out?